src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com/"
+      method="post"
+      class="search-container"
+      data-qa="keypress"
+    >
+      <label
+        data-qa="big"
+        class="label-search"
+      >
+        <img
+          class="search-icon"
+          src="./images/Search.svg"
+          alt="seach-icon"
+        >
+        <input
+          type="search"
+          name="seach"
+          id="seach-big"
+          class="search-field big-font"
```

Comment:
fix the classes according to the checklist  <img width="1106" alt="Screenshot 2022-11-09 at 15 12 20" src="https://user-images.githubusercontent.com/104769168/200839413-d2daa9a0-2adc-4e51-a8e2-b28e0cb2c3f3.png">